dex/pair/src/fee.rs

```diff
@@ -30,19 +33,17 @@ pub trait FeeModule:
     }
 
     #[endpoint(whitelist)]
-    fn whitelist_endpoint(&self, address: ManagedAddress) -> SCResult<()> {
-        self.require_permissions()?;
+    fn whitelist_endpoint(&self, address: ManagedAddress) {
+        self.require_permissions();
         let is_new = self.whitelist().insert(address);
-        require!(is_new, "ManagedAddress already whitelisted");
-        Ok(())
+        assert!(self, is_new, ERROR_ALREADY_WHITELISTED);
     }
 
     #[endpoint(removeWhitelist)]
-    fn remove_whitelist(&self, address: ManagedAddress) -> SCResult<()> {
-        self.require_permissions()?;
+    fn remove_whitelist(&self, address: ManagedAddress) {
+        self.require_permissions();
         let is_removed = self.whitelist().remove(&address);
-        require!(is_removed, "ManagedAddresss not whitelisted");
-        Ok(())
+        assert!(self, is_removed, ERROR_NOT_WHITELISTED);
     }
```

Comment:
Same here, simply remove, as trying to remove an element that does not exist does nothing.

Reply:
RV Suggested this behavior is better